addon/metrics-adapters/mixpanel.js

```diff
@@ -58,6 +58,11 @@ export default BaseAdapter.extend({
     }
   },
 
+  trackPage(options = {}) {
+    options.event = 'Page Viewed';
```

Comment:
We should do something like this instead:

```js
const pageView = { event: 'Page Viewed' };
const mergedOptions = merge(pageView, options);

this.trackEvent(mergedOptions);
```

That way, the `trackPage` call can still override the event name if needed.